CONTRIBUTION.md

```diff
@@ -0,0 +1,82 @@
+### Recommended development flow
+
+The most natural way to develop che-machine-exec it's using Eclipse CHE workspace.
+To install Eclipse CHE you need to choose infrastructure(openshift, kubernetes or docker) and set up it.
```

Comment:
I think that we not support docker as platform now

Reply:
maybe provide links to https://www.eclipse.org/che/docs/che-7/che-quick-starts.html#running-che-locally_che-quick-starts (how to install Che)
`Eclipse CHE` --> `Eclipse Che`

Reply:
>I think that we not support docker as platform now

> maybe provide links to https://www.eclipse.org/che/docs/che-7/che-quick-starts.html#running-che-locally_che-quick-starts (how to install Che)
Eclipse CHE --> Eclipse Che

 Done